Source/Csla.Blazor/State/StateManager.cs

```diff
@@ -10,59 +10,100 @@
 
 namespace Csla.Blazor.State
 {
-  /// <summary>
-  /// Get and save state from Blazor pages.
-  /// </summary>
-  /// <param name="sessionManager"></param>
-  public class StateManager(ISessionManager sessionManager)
-  {
-    private readonly ISessionManager _sessionManager = sessionManager;
-
     /// <summary>
-    /// Get state from cache.
+    /// Get and save state from Blazor pages.
     /// </summary>
-    public Task InitializeAsync()
+    /// <param name="sessionManager">The session manager to use for state management.</param>
+    public class StateManager(ISessionManager sessionManager)
```

Comment:
Wrong formatting. 3 spaces instead of 2 within the methods.
Looks like this is true for the whole file.

Reply:
now I runned the dotnet format csla.test.sln and I got more than 300 files changed, I think that after this merge we should open a new PR to format them all, what do you think @StefanOssendorf ?